src/core/inlines.js

```diff
@@ -129,20 +129,16 @@ export function run(conf) {
     if (subtxt.length === 1) continue;
 
     const df = document.createDocumentFragment();
-    while (subtxt.length) {
-      const t = subtxt.shift();
+    for (let i = 0; i < subtxt.length; i++) {
```

Comment:
Is there some reason we can't do `for (const t of  subtext) {`?

Reply:
Hmm yeah, we don't even need an exact index number. Maybe just:

```js
let match = true;
for (const t of subtext) {
  match = !match;
  if (match) {
    df.appendChild(document.createTextNode(t));
  } else if (t.startsWith(...)) {
    // ...
  }
}